app/scripts/modules/core/src/search/global/GlobalSearchRecentItems.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { UISref } from '@uirouter/react';
+
+import { ISearchResultPodData } from 'core/search/infrastructure/SearchResultPods';
+import { SearchResult } from 'core/search/infrastructure/SearchResult';
+
+export interface IGlobalSearchRecentItemsProps {
+  categories: ISearchResultPodData[];
+  onItemKeyDown: (event: React.KeyboardEvent<HTMLAnchorElement>) => any;
+  onResultClick: Function;
+  resultRef: (categoryIndex: number, resultIndex: number, ref: HTMLAnchorElement) => any;
+}
+
+export function GlobalSearchRecentItems({
+  categories,
+  onItemKeyDown,
+  onResultClick,
+  resultRef
```

Comment:
I love trailing commas

Reply:
![ohgodwhy](https://bukk.it/waitwhatohgodno.gif)

Reply:
same,